Sources/AblyChat/Room.swift

```diff
@@ -115,6 +136,15 @@ internal actor DefaultRoom<LifecycleManagerFactory: RoomLifecycleManagerFactory>
         try await lifecycleManager.performDetachOperation()
     }
 
+    internal func release() async {
+        await lifecycleManager.performReleaseOperation()
+
+        // CHA-RL3h
+        for channel in channels.values {
+            realtime.channels.release(channel.name)
```

Comment:
I know this is merged, but still leaving it here. Shouldn't this be inside `performReleaseOperation`?

Reply:
I think that I thought that since the `Room` is the one who fetches the channels, it made sense for it to also be the one who releases them 🤷, not a very strongly held belief though